_posts/2024-03-22-kraft-migration.md

```diff
@@ -0,0 +1,388 @@
+---
+layout: post
+title:  "From ZooKeeper to KRaft: let's automate the migration with Strimzi"
+date: 2024-03-22
+author: paolo_patierno
+---
+
+The Apache Kafka project has been using [Apache ZooKeeper](https://zookeeper.apache.org/) since its inception to store metadata.
```

Comment:
```suggestion
The Apache Kafka project has been using [Apache ZooKeeper](https://zookeeper.apache.org/) since its inception to store metadata, handle leader election, and broker discovery.
```

Reply:
Isn't this suggestion similar to the sentence coming on the next line?

> Registered brokers, the current controller and topics' configuration are just a few of the data that ZooKeeper stores for supporting a running Kafka cluster.

Reply:
Yeah, maybe we can merge the two. I think it's important to highlight that ZK is not only a metadata store, but much more.